### PR TITLE
Replace manual cert recovery with automated one

### PR DESCRIPTION
**Pull request by bergerhoffer**
https://issues.redhat.com/browse/OSDOCS-743

Replaces #22272.

Preview: https://pr-22272-auto-recovery-update--ocpdocs.netlify.app/openshift-enterprise/latest/backup_and_restore/disaster_recovery/scenario-3-expired-certs.html

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@tnozicka I overtook your PR to make some minor adjustments. Let me know if this looks good to you, thanks! 

FYI @xingxingxia this is the new PR, with just some minor adjustments, in case you have any additional feedback. Thanks!

**Review by tnozicka — APPROVED** [review]
/lgtm
thanks

File: modules/dr-recover-expired-control-plane-certs.adoc [hunk 1]
```diff
@@ -5,167 +5,33 @@
...
 = Recovering from expired control plane certificates
 
-Follow this procedure to recover from a situation where your control plane certificates have expired.
+As of {product-title} 4.5, the cluster can automatically recover from expired control plane certificates. You no longer need to perform the manual steps that were required in previous versions.
 
-.Prerequisites
+The exception is that you must manually approve the pending `node-bootstrapper` certificate signing requests (CSRs) to recover kubelet certificates.
 
-* SSH access to master hosts.
+Use the following steps to approve the pending `node-bootstrapper` CSRs:
```
Comment (mburke5678) [inline modules/dr-recover-expired-control-plane-certs.adoc:12]: Should this come after the `.Procedure`? If not, s/CSRs:/CSRs.?

Comment (mburke5678) [reply modules/dr-recover-expired-control-plane-certs.adoc:12]: Do the `node-bootstrapper`CSRs have `node-bootstrapper` in the name?

File: modules/dr-recover-expired-control-plane-certs.adoc [hunk 2]
```diff
@@ -5,167 +5,33 @@
...
-----
-
-.. Delete stale kubelet data.
-+
-----
-# rm -rf /var/lib/kubelet/pki /var/lib/kubelet/kubeconfig
-----
-
-.. Restart the kubelet.
+. Review the details of a CSR to verify that it is valid:
```
Comment (mburke5678) [inline modules/dr-recover-expired-control-plane-certs.adoc:24]: How does the user know if the CSR is valid?

**Comment by mburke5678** [comment]
@bergerhoffer A few questions. Otherwise LGTM

File: modules/dr-recover-expired-control-plane-certs.adoc (hunk 1 above)
Comment (bergerhoffer) [reply modules/dr-recover-expired-control-plane-certs.adoc:12]: Yes they do

**Comment by xingxingxia** [comment]
@bergerhoffer , LGTM, thanks!

**Comment by bergerhoffer** [comment]
Thanks @xingxingxia!

@tnozicka We're getting this in for 4.5, but if this gets backported to 4.4, let me know so that we can make an update there whenever it is time. Thanks!

**Pull request merged**